l10n/Makefile

```diff
@@ -0,0 +1,10 @@
+LANG_FILES=$(shell find . -name "*.xml")
+LANG_DESTS=../java/l10n/$(LANG_FILES:.xml=.json)
+
+$(LANG_DESTS): $(LANG_FILES)
```

Comment:
There's no need to put this rule into a separate Makefile, and [Recursive Make Considered Harmful](http://aegis.sourceforge.net/auug97.pdf), so I would put this into the top-level Makefile, even though it takes files from l10n/ as input and generates output in java/.